doc/Doxyfile

```diff
@@ -468,8 +468,7 @@ INPUT                  = $(DOX_INPUT)
 # *.c *.cc *.cxx *.cpp *.c++ *.java *.ii *.ixx *.ipp *.i++ *.inl *.h *.hh *.hxx 
 # *.hpp *.h++ *.idl *.odl *.cs *.php *.php3 *.inc *.m *.mm *.py
 
-FILE_PATTERNS          = *.h \
-                         *.cc
+FILE_PATTERNS          = *.h
```

Comment:
in .cc files are no public documentation, but template spezialisations and Macros, which Doxygen can't handle